### ISSUE TITLE
ERROR: Git repository not found at 'CPLEX'

### ISSUE THREAD

**Issue by IgnacioRubioScola** [JuliaOpt/CPLEX.jl#221]
On Julia 1.1
```
(v1.1) pkg> add CPLEX
[ Info: resolving package specifier `CPLEX` as a directory at `~/CPLEX`.
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
   Cloning git-repo `CPLEX`
ERROR: Git repository not found at 'CPLEX'
```

**Comment by IgnacioRubioScola** [comment]
doing:
```
using Pkg
Pkg.add("CPLEX")
```
works fine

**Comment by odow** [comment]
Great that you got it working. Please re-open if it is a problem. It doesn't look CPLEX-specific, maybe a weird Pkg bug? If you can reproduce on other packages, you could ask a question on https://discourse.julialang.org/.

**Comment by henriquebecker91** [comment]
I had the same problem today. Using "Pkg.add" instead of "] add" has worked for me. I was using Julia 1.1 too. I do not think it needs to be reopened.

**Comment by odow** [comment]
If you can reproduce this on, you could post something on the forum. But it will probably only be fixed if you can reproduce on Julia 1.3.

**Comment by jonathanBieler** [comment]
I think the issue happens when you have folders with the same name as the package you want to install in your current directory, you can reproduce the issue like this :

```julia
shell> cd ~/.julia/packages/

pkg> add Distributions
[ Info: resolving package identifier `Distributions` as a directory at `~/.julia/packages/Distributions`.
   Cloning git-repo `Distributions`
ERROR: Git repository not found at 'Distributions'
```

Not sure if that should be reported in Pkg, it's not a bug, but the error message is a bit obscure, and the different behaviour of `Pkg.add` is also puzzling.
